challenges/fifo_animal_shelter/fifo_animal_shelter.py

```diff
@@ -0,0 +1,68 @@
+class AnimalShelter():
+
+    def __init__(self):
+        self.headCat = None
+        self.headDog = None
+        self.tailCat = None
+        self.tailDog = None
```

Comment:
It looks like you are using these as two queues. Instead of reimplementing a queue, you can just use the Queue class that you already built.